README.md

```
@@ -0,0 +1,32 @@
+</p>
+
+
+# Athena [![GitHub release][release-image]][release-url] [![made-with-python][python-image]][python-url]
+
+Athena is a tool to generate coverage statistics for NGS data, and combine these into an interactive HTML report. This gives both summary level and in depth information as to the coverage of the data, including various tables and plots to visualise the data. Examples of the output statistics files and [report][report-link] may be found in `data/example`.
+
+
+## Usage
+
+
+## <img src="athena/data/images/moby.png" width="34"/> Docker
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)